src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -561,27 +561,54 @@ Where:
 - `elasticsearch.client` choose the RestClient protocol
 - `elasticsearch.addressList` is a comma separated list of URL.
 
-#### Advanced REST Client configuration
-
-If you need to use Basic Authentication or X509 certificate you need to override the default template,
-Here are all the available options:
-```
-  <extension target="org.nuxeo.elasticsearch.ElasticSearchComponent" point="elasticSearchClient">
-  <elasticSearchClient class="org.nuxeo.elasticsearch.client.ESRestClientFactory">
-    <option name="addressList">https://somenode:443</option>
-    <!-- basic auth -->
-    <option name="username">scott</option>
-    <option name="password">tiger</option>
-    <!-- use a keystore -->
-    <option name="keystore.path">/usr/lib/jvm/java-8-oracle/jre/lib/security/cacerts</option>
-    <option name="keystore.password">changeit</option>
-    <!-- connection and socket timeout for the rest client -->
-    <option name="connection.timeout.ms">5000</option>
-    <option name="socket.timeout.ms">20000</option>
-  </elasticSearchClient>
-  </extension>
+#### Advanced REST Client configuration with X-Pack
```

Comment:
Maybe add a new section for X-Pack, because there are also users interested with AWS elasticsearch or other custom elasticsearch deployment behind nginx.